metis/database/client.py

```
@@ -44,7 +44,7 @@ class ArangoHttp2Config:
     """Configuration for the HTTP/2 client."""
 
     database: str = "_system"
-    socket_path: str = "/tmp/arangodb.sock"
+    socket_path: str | None = "/tmp/arangodb.sock"  # None = use TCP
```

Ruff (0.13.3)

Probable insecure usage of temporary file or directory: "/tmp/arangodb.sock"

(S108)